src/gui/guiChatConsole.h

```diff
@@ -138,4 +142,6 @@ class GUIChatConsole : public gui::IGUIElement
 	bool m_cache_clickable_chat_weblinks;
 	// Track if a ctrl key is currently held down
 	bool m_is_ctrl_down;
+	// Render a scrollbar (mainly for Android)
+	bool m_scrollbar_enabled;
```

Comment:
this separate field is not needed, you can just check for whether `m_scrollbar` is non-null instead